backend/chainlit/llama_index/callbacks.py

```diff
@@ -154,7 +154,7 @@ def on_event_end(
 
             token_count = self.total_llm_token_count or None
             raw_response = response.raw if response else None
-            model = raw_response.get("model", None) if raw_response else None
+            model = raw_response.model if hasattr(raw_response, "model") else None
```

Comment:
I would suggest changing this to `getattr(model, "mode", None)`.

It technically does the same thing but is more common and readable and thus maintainable.

Reply:
Thank you, TIL! Updated the PR